neo/Consensus/ConsensusService.cs

```diff
@@ -92,14 +103,15 @@ private void CheckExpectedView(byte view_number)
             }
         }
 
-        private void CheckSignatures()
+        private void CheckPreparations()
         {
-            if (context.Signatures.Count(p => p != null) >= context.M && context.TransactionHashes.All(p => context.Transactions.ContainsKey(p)))
+            if (context.Preparations.Count(p => p) >= context.M && context.TransactionHashes.All(p => context.Transactions.ContainsKey(p)))
```

Comment:
Should we count p only when `p == true` or `p != null` because `context.Preparations` is initialized as 
`Preparations = new bool[Validators.Length];`